ExtraDry/ExtraDry.Blazor/Models/HyperLinkInfo.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace ExtraDry.Blazor.Models;
+
+/// <summary>
+/// Represents a hyperlink which wraps a method call and additional information about how to present
+/// the hyperlink using the method's signature.
+/// </summary>
+public partial class HyperlinkInfo
+{
+    /// <summary>
+    /// Create a `HyperLinkInfo` with a reference to the ViewModel it will execute on and the method to call.
+    /// </summary>
+    public HyperlinkInfo(object viewModel, MethodInfo method)
+    {
+        ViewModel = viewModel;
+        Method = method;
+        Initialize(method);
+    }
+
+    /// <summary>
+    /// The reflected method for the command.
+    /// </summary>
+    public MethodInfo Method { get; set; }
+
+    /// <summary>
+    /// The view model that this hyperlink is defined as being part of.
+    /// Used by `ExecuteAsync` to invoke the command on the correct object instance.
+    /// </summary>
+    public object ViewModel { get; set; }
+
+    /// <inheritdoc cref="HyperlinkAttribute.PropertyName" />
+    public string? PropertyName{ get; set; }
+
+    public HyperlinkContext? Execute(object? arg = null)
+    {
+        return Method.Invoke(ViewModel, new object?[] { arg }) as HyperlinkContext;
```

Comment:
As we're in a Blazor component, these types of issues would be nice to have a Console warning allowing developers to catch bugs earlier.  Maybe one for hyperlink was null, another for Hyperlink method returned wrong data type?  And another for wrong signature?  E.g. if Wayne were to experimentally do the following:
```
   [Hyperlink("Property")]
   public string GetHyperlink() => "/destination/url";
```
How do the error messages guide him to the correct implementation?